src/views/InvolvementProfile/InvolvementProfile.module.scss

```diff
@@ -11,17 +11,3 @@
 .update_image_components {
   margin-top: 15px;
 }
-
-.dialogContentText {
-  text-align: 'center';
-}
-
-.imageDrop {
-  max-width: 320px;
-  max-height: 320px;
-  border-radius: 0.5rem;
-}
-
-.activityURL {
-  font-weight: bold;
```

Comment:
These classes are still being used in the index file, did you mean to delete them?